oauth2-api/src/main/java/net/oauth2/AccessTokenGrantRequest.java

```diff
@@ -87,6 +86,9 @@ public String getClientSecret() {
 	@Override
 	public Map<String, Object> map() throws Exception {
 		Map<String, Object> grant = BeanUtils.asMap(this, getPropertyMap());
+		if(this.scope !=null){
+			grant.put("scope",String.join(" ", this.scope));
+		}
```

Comment:
Map the scopes to a single space separated string.   No unit test for this existed previously.